### ISSUE TITLE
ITUNES

### ISSUE THREAD

**Issue by PANOSSS1980** [GeorgiSwiftDeveloper/Wave_Music#1]
MY FRIEND PLEASEEEEEEEE CAN YOU MAKE A WEB VERSION?I HAVE NO IDEA FROM DEVELOPING AND I DONT KNOW HOW TO DO IT!!!I AM BEGGING YOU MAN

**Comment by GeorgiSwiftDeveloper** [comment]
Hi, I don't really understand what is the Web version ? Could you please clarify . Thank you

**Comment by PANOSSS1980** [comment]
hello from Greece!!!yes,of course my friend and thanks so much where answer me!!!this is for searching music...albums and songs from itunes...right?

**Comment by GeorgiSwiftDeveloper** [comment]
Yes here u can search music from the iTunes as part of that you can see upcoming movies, this app just created few days ago. And I need to modify this. This is just a sample.

**Comment by PANOSSS1980** [comment]
woowwwwwwwwww....my friend can you release it like web page?it would be greattttttttttttttt

**Comment by GeorgiSwiftDeveloper** [comment]
This is just a coding challenge ) If u new in programming I  will suggest you learn Swift and Objective-C languages.

**Comment by PANOSSS1980** [comment]
my good friend I have noooooooooooo idea from developing and programming!!!unfortunately!!!just I speak spanish very well and I love spanish music so much!!!thats why I search for one web where use itunes api for songs and albums but with unlimited results

**Comment by PANOSSS1980** [comment]
my friend pleaseeeeeeeee can you upload all the commands like zip and download and write just the commands on cmd to run it?because I  have no idea how to make it!!!
